keys/storage/filestorage/main_test.go

```diff
@@ -68,7 +69,7 @@ func TestBasicCRUD(t *testing.T) {
 	assert.NotNil(err)
 
 	// and then Get and List don't work
-	_, _, err = store.Get(name)
+	_, _, _, err = store.Get(name)
```

Comment:
Now that the code is separated, maybe useful to test the
read(write(x)) and
readInfo(writeInfo(x)) pairs
to make sure they return the original info

A bit lower level test